open-api/lang-parser/src/jjtree/java/io/deephaven/lang/generated/Chunker.jj

```diff
@@ -652,10 +961,20 @@ checkInterrupts();
     } ) // optional semi-colon delimiter
    } catch (ParseException e) {
      // Whenever there is an unhandled parse exception,
-     // gobble up the rest of the line and attach to previous node.
+     // gobble up the rest of the line and attach to previous node...
+     // unless the current item on the stack ends w/ a valid statement terminator
      if (expr != null) {
-       token = expr.addJunk(eatJunk());
-       fixState(token);
+       Token tok = expr.jjtGetLastToken();
+//       switch(tok.kind) {
```

Comment:
should we have todos, issues filed for decls like this that just emit already-commented-out code? or is this intended to be very temporary, already fixed in another patch sort of thing, or something like "this is how you'd think it should be done, but that's not correct because (missing comment explanation)"?

Reply:
It was an attempt to fix a proble that was solved another way. deleted.